script/Ciel/Day 8/08_01_CIEL08_4_1.txt

```diff
@@ -228,6 +228,7 @@ Hold on, we're meeting here again tomorrow, right?"
 [sha:0087e8a0787a6714fb0dde642bb0eb02a8d1ee09]{
 -- Page 405, Offset 19272.
 -- 「もっちろん！
+// Spruceable?
```

Comment:
maybe "Definitely!" would fit the tone better?

Reply:
Sure, why not.